taskcluster/android_taskgraph/gradle.py

```diff
@@ -7,6 +7,11 @@ def get_gradle_project(task):
     attributes = task.get("attributes", {})
     gradle_project = attributes.get("component")
     if not gradle_project:
+        shipping_product = attributes.get("shipping-product", "")
```

Comment:
`shipping-product` is a concept used in gecko. It enables us to support both `devedition` and `firefox`. We should leverage this feature here too. We technically have at least to products: `focus` and `fenix`. 

This change is incomplete. There are other places where we should use `shipping-product` instead. However, this patch is blocking PRs from landing because we don't run UI tests there anymore. That's why I just handled the UI tests.

Reply:
Thanks for the info!

Reply:
shipping-product in gecko is used in release graphs (per https://firefox-source-docs.mozilla.org/taskcluster/attributes.html#shipping-product), it seems weird to use that same attribute for tests IMO.

Reply:
I agree `shipping-product` was originally defined to be used in release promotion. That said, it's also used in some tests in gecko[1]. I think it makes sense to use this attribute because we won't have to define another one to carry the same data. 

[1] https://searchfox.org/mozilla-central/rev/6444ed17e9f5e0d4e7dabc13c930d46b529fca15/taskcluster/ci/startup-test/kind.yml#54

Reply:
startup-test in gecko is special, since it blocks release tasks, it's actually part of releases, in a sense.

Reply:
So I guess my point is we're not using the same attribute as gecko here, we're reusing the attribute name to mean something different, and IMO that's worse than coming up with a new name for it.